app/javascript/packs/homePage.jsx

```diff
@@ -23,15 +23,12 @@ function renderTagsFollowed(user = userData()) {
   const tagsFollowedContainer = document.getElementById(
     'sidebar-nav-followed-tags',
   );
-  if (!tagsFollowedContainer) {
-    // Not on the homepage, so nothing to do.
+  if (user === null || !tagsFollowedContainer) {
+    // Return and do not render if the user is not logged in
+    // or if this is not the home page.
     return false;
   }
-  if (user === null || document.getElementById('followed-tags-wrapper')) {
-    return;
-  }
```

Comment:
@nickytonline This is the change that runs a risk of reintroducing the bug from the hotfix mentioned in the PR description.

[Relevant Slack thread](https://forem-team.slack.com/archives/C7GE84DEJ/p1584534941287100)